### PR TITLE
New Solid Refresh

### PR DESCRIPTION
**Pull request by lxsmnsyc**
This PR is in correspondence with [the new Solid Refresh](https://github.com/solidjs/solid-refresh/pull/34)

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    console.log('Unmounted Decrement');
+  });
+  return (
+    <button onClick={counter.decrement}>
+      Decrement
+    </button>
+  );
+}
+
+export default function App() {
```
Comment (lxsmnsyc) [inline example/src/App.tsx:50]: This minimal example is to test the following:
- Context lookup is working in HMR
- Granular mode test

**Comment by ryansolid** [comment]
OK, I've released solid-refresh. What's next?

**Comment by lxsmnsyc** [comment]
Sorry for that

**Comment by ryansolid** [comment]
Thank you

**Pull request merged**